src/format/pdf/format-pdf.ts

```diff
@@ -364,12 +364,17 @@ function pdfLatexPostProcessor(
       // Replace notes with side notes
       lineProcessors.push(sideNoteLineProcessor());
     }
-
     lineProcessors.push(captionFootnoteLineProcessor());
-    if (format.metadata[kCodeAnnotations] !== false) {
+
+    if (
+      format.metadata[kCodeAnnotations] as boolean !== false &&
+      format.metadata[kCodeAnnotations] as string !== "none"
+    ) {
+      console.log("NOT HERE");
```

Comment:
Extra output

Reply:
"one of us, one of us, one of us"

(I'll show myself out.)

Reply:
😆 Last PR of the day,  right before going out to get the kids home 
I know I shouldn't do that in a hurry ! 🤦‍♂️ 😅